libs/vm-modules/tests/unit/ml/dataloader_tests.cpp

```diff
@@ -0,0 +1,77 @@
+//------------------------------------------------------------------------------
+//
+//   Copyright 2018-2020 Fetch.AI Limited
+//
+//   Licensed under the Apache License, Version 2.0 (the "License");
+//   you may not use this file except in compliance with the License.
+//   You may obtain a copy of the License at
+//
+//       http://www.apache.org/licenses/LICENSE-2.0
+//
+//   Unless required by applicable law or agreed to in writing, software
+//   distributed under the License is distributed on an "AS IS" BASIS,
+//   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//   See the License for the specific language governing permissions and
+//   limitations under the License.
+//
+//------------------------------------------------------------------------------
+
+#include "vm_modules/ml/dataloaders/dataloader.hpp"
+#include "vm_modules/ml/utilities/conversion_utilities.hpp"
+#include "vm_modules/vm_factory.hpp"
+
+#include "gmock/gmock.h"
+
+namespace {
```

Comment:
why not put all of it inside an anonymous namespace?

Reply:
Fair point!